src/components/LandingPage/LandingPage.js

```diff
@@ -1,11 +1,51 @@
-import React from "react";
+import background from "./bg-landingPage.jpg";
+import { Card } from "react-bootstrap";
+import styles from "./LandingPage.module.css";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 const LandingPage = () => {
-  return (
-    <div style={{ height: "100vh" }}>
-      <div>LandingPage</div>
-    </div>
-  );
+	return (
+		<>
```

Comment:
You don't need to use a fragment when you have a parent element.

Reply:
Ok, I'm already fixing it